hw/ip/rv_timer/dv/rv_timer_sim_cfg.hjson

```diff
@@ -46,14 +46,7 @@
   tests: [
     {
       name: rv_timer_smoke
```

Comment:
This would be `rv_timer_random`. Note that there is no requirement mandating an IP to have `<ip>_smoke` test specification defined.  The only mandatory thing is `smoke` regression suite, for which the user chooses the tests.